src/components/App.js

```diff
@@ -0,0 +1,71 @@
+import '../App.css';
+import CampaignTable from './CampaignTable';
+import Sidebar from './Sidebar';
+import React, { useEffect, useState } from 'react';
+import {
+    BrowserRouter as Router,
+    Routes,
+    Route,
+    useNavigate
+} from "react-router-dom";
+
+export default function App() {
+  const [campaigns, setCampaigns] = useState([]);
+
+  useEffect(() => {
+    fetch(`http://localhost:3000/api/campaigns`)
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error('Network response was not ok');
+        }
+        return response.json();
+      })
+      .then((data) => {
+        setCampaigns(data);
+      })
+      .catch((error) => {
+        console.error('Error fetching campaigns:', error);
+      });
+  }, []);
+
+  const draftCampaigns = campaigns.filter((campaign) => campaign.status === false);
```

Comment:
Why you took the decision to filter the campaigns at the frontend and not at the backend ?